api/draft_registrations/views.py

```diff
@@ -50,9 +50,9 @@ def check_resource_permissions(self, resource):
 
 class DraftRegistrationList(NodeDraftRegistrationsList):
     permission_classes = (
-        IsContributorOrAdminContributor,
         drf_permissions.IsAuthenticatedOrReadOnly,
         base_permissions.TokenHasScope,
+        DraftRegistrationPermission,
```

Comment:
This is still a noop, since DraftRegistrationPermission only defines `has_object_permission` -- and this endpoint doesn't need any extra permissions, since it's just surfacing the current user's drafts (https://media1.tenor.com/m/j5TI9U8hISQAAAAC/emperors-new-groove-why.gif)

Reply:
Without it, a bunch of our tests are failing because it is letting unauthorized users create draft registrations.

The following tests failed when it was removed:

test_write_only_contributor_cannot_create_draft
test_read_only_contributor_cannot_create_draft
test_draft_registration_attributes_copied_from_node
test_logged_in_non_contributor_cannot_create_draft